vyper/functions/convert.py

```diff
@@ -52,7 +64,7 @@ def to_int128(expr, args, kwargs, context):
         else:
             raise InvalidLiteral(f"Unknown numeric literal type: {in_arg}")
 
-    elif input_type == "bytes32":
+    elif input_type in ("bytes32", "int256"):
```

Comment:
This seems weird

Reply:
The logic is identical for converting both types - it just applies a bounds check.